package/android/src/oldarch/java/com/reactnativecommunity/slider/ReactSliderManager.java

```diff
@@ -28,11 +28,23 @@ public class ReactSliderManager extends SimpleViewManager<ReactSlider> {
           new SeekBar.OnSeekBarChangeListener() {
             @Override
             public void onProgressChanged(SeekBar seekbar, int progress, boolean fromUser) {
+              ReactSlider slider = (ReactSlider)seekbar;
+
+              if(progress < slider.getLowerLimit()) {
+                progress = slider.getLowerLimit();
+                seekbar.setProgress(progress);
+              }
+
+              if(progress > slider.getUpperLimit()) {
+                progress = slider.getUpperLimit();
+                seekbar.setProgress(progress);
+              }
```

Comment:
Same as [the change above](https://github.com/callstack/react-native-slider/pull/432/files#diff-dc644f156dbc97ab8c8e17dcd152efba4e8b761399400b27eddba3a87640521bR43) but for the old arch